scripts/build_gh_pages

```diff
@@ -29,25 +29,38 @@ if [[ -n "$(ls "${TARGET_DIR}"/*)" ]]; then
   exit 1
 fi
 
-# Generate docs directly in the target dir.
-cargo doc --no-deps --target-dir="${TARGET_DIR}"
+readonly CURRENT_DIR="$(pwd)"
 
-# Remove non-doc artifacts from the target dir.
-rm --recursive --force "${TARGET_DIR}/debug"
+paths=("." "oak_abi" "oak_utils")
+names=("oak" "oak_abi" "oak_utils")
+titles=("SDK" "ABI" "Utils")
 
-# Remove non-deterministic files.
-rm "${TARGET_DIR}/.rustc_info.json"
+for i in {0..2}; do
```

Comment:
Maybe use the length of one of the arrays rather than `2` so there's one fewer thing to keep in sync?